siliconcompiler/tools/vivado/__init__.py

```diff
@@ -0,0 +1 @@
+from .vivado import *
```

Comment:
I see the reason for this, but I think you should do explicit imports so it's clear what exactly it is you want